2025/timed-text-wg.html

```diff
@@ -227,7 +227,7 @@ <h3>New Normative Specifications</h3>
 
   </dl>
 
-  <h3>New Normative Specifications Continuously Under Development</h3>
+  <h3>Normative Specifications Continuously Under Development</h3>
   <p>The Working Group intends to develop the following new W3C normative specifications:</p>
```

Comment:
Shouldn't this line change too?
```suggestion
  <p>The Working Group intends to continue developing the following W3C normative specifications:</p>
```